codice/ddf
# DDF-1848: removed old maven module and quit ignoring resources directory.

#### What does this PR do?

* removes an unused `features.xml` file
* enable tests for `libs-pomfix` module, got removed by accident I think
* updates some poms with missing dependencies
* updates `pom-fix` to ignore only its project directory instead of the more general `resources ` directory.
  * this was causing `pom-fix` to ignore all `features.xml` files, not just the test resources for the `lib-pomfix` module.

#### Who is reviewing it?

@stustison 
@lessarderic 
@rzwiefel 

#### How should this be tested?

Just pull down the PR and only `mvn clean install` the `libs-pomfix` and `libs-pomfix-run` maven modules. Make sure to clean the `node_modules` directory in the `libs-pomfix-run` module.

#### What are the relevant tickets?

https://codice.atlassian.net/browse/DDF-1848
